kernel/src/transaction/mod.rs

```diff
@@ -15,6 +16,19 @@ use crate::{
 
 use url::Url;
 
+/// The schema that the [`Engine`]'s [`ParquetHandler`] is expected to use when reporting information about
```

Comment:
probably should update docstring on the `add_files_schema()` method which is basically the public interface to this const